extra/patterns/patterns.js

```diff
@@ -0,0 +1,18 @@
+var arg = process.argv
+
+function equilateral(size){
+    if(size>=2 && size <=10){  // Condition to check if the input argument is within the range.
+       for(var i=1;i<=size;i++){
```

Comment:
var should not be used. Use let or const only

Reply:
Suggested changes implemented successfully.